src/main/java/com/nike/cerberus/operation/composite/CreateCerberusEnvironmentOperation.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.operation.composite;
+
+import com.nike.cerberus.command.cms.CreateCmsClusterCommand;
+import com.nike.cerberus.command.cms.CreateCmsCmkCommand;
+import com.nike.cerberus.command.cms.CreateCmsConfigCommand;
+import com.nike.cerberus.command.composite.CreateCerberusEnvironmentCommand;
+import com.nike.cerberus.command.core.CreateBaseCommand;
+import com.nike.cerberus.command.core.CreateDatabaseCommand;
+import com.nike.cerberus.command.core.CreateEdgeDomainRecordCommand;
+import com.nike.cerberus.command.core.CreateLoadBalancerCommand;
+import com.nike.cerberus.command.core.CreateRoute53Command;
+import com.nike.cerberus.command.core.CreateSecurityGroupsCommand;
+import com.nike.cerberus.command.core.CreateVpcCommand;
+import com.nike.cerberus.command.core.CreateWafCommand;
+import com.nike.cerberus.command.core.GenerateCertsCommand;
+import com.nike.cerberus.command.core.UploadCertFilesCommand;
+import com.nike.cerberus.domain.environment.Stack;
+
+import java.util.List;
+
+/**
+ * Operation class for CreateCerberusEnvironmentCommand
+ */
+public class CreateCerberusEnvironmentOperation extends CompositeOperation<CreateCerberusEnvironmentCommand> {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected List<ChainableCommand> getCompositeCommandChain() {
+        return ChainableCommandListBuilder.create()
```

Comment:
Seems like `Lists.newArrayList()` would accomplish the same.

Reply:
Yes you could but then you would have to type more stuff, I was liking the builder pattern because it involved typing less cruft.

    List<ChainableCommand> commandList = new LinkedList()<>;
    commandList.add(new Command());
    commandList.add(new Command());
    commandList.add(new Command());
    commandList.add(new Command());
    commandList.add(new Command());
    return commandList;

vs

    return Builder.create()
          .add(new Command())
          .add(new Command())
          .add(new Command())
          .add(new Command())
          .add(new Command())
          .add(new Command())
          .build

Reply:
I mean like this:
`
Lists.newArrayList(
   new Command(),
   new Command(),
   new Command(),
)
`

Reply:
Got it, yeah thats better. I will update

Reply:
- [ ] Done.